docs/source/en/guides/integrations.md

```diff
@@ -280,6 +276,152 @@ class PyTorchModelHubMixin(ModelHubMixin):
 
 And that's it! Your library now enables users to upload and download files to and from the Hub.
 
+### Advanced usage
+
+In the section above, we quickly discussed how the [`ModelHubMixin`] works. In this section, we will see some more advanced features the mixin offers to improve the integration of your library with the Hugging Face Hub.
+
+#### Model card
+
+[`ModelHubMixin`] generates the model card for you. Model cards are files that accompany the models and provide handy information. Under the hood, model cards are simple Markdown files with additional metadata. Model cards are essential for discoverability, reproducibility, and sharing! Check-out the [Model Cards guide](https://huggingface.co/docs/hub/model-cards) for more details.
```

Comment:
Probably more "important" than "handy" which gives the impression it is just for convenience :)

```suggestion
[`ModelHubMixin`] generates the model card for you. Model cards are files that accompany the models and provide important information about them. Under the hood, model cards are simple Markdown files with additional metadata. Model cards are essential for discoverability, reproducibility, and sharing! Check out the [Model Cards guide](https://huggingface.co/docs/hub/model-cards) for more details.
```